src/main/utils/channel/channel-listeners/enlist-message-channel-listener.injectable.ts

```diff
@@ -16,13 +15,7 @@ const enlistMessageChannelListenerInjectable = getInjectable({
     const ipcMain = di.inject(ipcMainInjectable);
 
     return ({ channel, handler }) => {
-      const nativeOnCallback = (_: IpcMainEvent, message: unknown) => {
-        pipeline(
-          message,
-          tentativeParseJson,
-          handler,
-        );
-      };
+      const nativeOnCallback = (_: IpcMainEvent, message: unknown) => toJS(handler(message));
```

Comment:
Nit: could use `pipeline`for left-to-right, top-to-bottom -readability.

```
pipeline(message, handler, toJS)
```

Reply:
For functions I prefer the reverse-Polish notation style that comes with nested brackets.